opm/autodiff/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -189,6 +190,53 @@ namespace Opm
         std::vector<int> current_controls_;
     };
 
+    static inline data::Wells report( const WellStateFullyImplicitBlackoil& state,
```

Comment:
The `static` is unnecessary and misleading (I first thought this was a static method of the well state class).